pkg/config/node.go

```diff
@@ -202,13 +202,25 @@ func GetConfig(kubeconfigPath, clusterConfigPath, resolvConfPath string, apiVip
 	}
 
 	vips := make([]net.IP, 0)
+	node.Cluster.APIVIPRecordType = "A"
+	node.Cluster.APIVIPEmptyType = "AAAA"
```

Comment:
Shouldn't these fields be added to the struct in line 26?

Reply:
Somebody forgot to commit his local changes...